wifiphisher/pywifiphisher.py

```diff
@@ -595,6 +595,10 @@ def start(self):
                        "rogue Access Point").format(G, W, ap_iface.get_name())
 
             kill_interfering_procs()
+            #randomize the adapters MAC addresses
+            self.network_manager.randomize_interface_mac(ap_iface)
+            self.network_manager.randomize_interface_mac(mon_iface)
+
             self.network_manager.set_interface_mode(mon_iface, "monitor")
```

Comment:
@sophron this is because even if user specifies `--nojamming` option, we still need to set the interface to monitor mode first, since we need to do ap selection.